Sail/src/Sail/resources/loaders/NotFBXLoader.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "Sail/graphics/geometry/Model.h"
+#include "Sail/graphics/geometry/Animation.h"
+#include <string>
+
+namespace NotFBXLoader {
+
+	void Load(std::string filename, Model*& model, Shader* shader, AnimationStack*& animationStack);
+	void Save(std::string filename, Model* model, AnimationStack* animationStack);
```

Comment:
filename should be const references to avoid extra allocation whenever SSO (short string optimization) is not possible.